pkg/get/tools.go

```diff
@@ -2272,5 +2272,44 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "hairyhenderson",
+			Repo:        "gomplate",
+			Name:        "gomplate",
+			Description: "A flexible commandline tool for template rendering. Supports lots of local and remote datasources.",
+			URLTemplate: `
+			{{ $os := "linux" }}
+			{{ $arch := "amd64" }}
+			{{ $ext := "" }}
+
+			{{- if eq .Arch "aarch64" -}}
+			{{ $arch = "arm64" }}
+			{{- else if eq .Arch "arm64" -}}
+			{{ $arch = "arm64" }}
+			{{- else if eq .Arch "armv7l" -}}
+			{{ $arch = "armv7" }}
+			{{- else if eq .Arch "armv6l" -}}
+			{{ $arch = "armv6" }}
+			{{- end -}}
```

Comment:
I don't think armv6l should be included since it is dropped for a lot of project nowdays and we started to exclude it too.

Reply:
Agreed. Please hit "Resolve conversation" if this has been done.